### PR TITLE
Fixes the overriding of config file

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by domcleal** [comment]
Would you mind also fixing rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec?

**Comment by ares** [comment]
Fixed, should I merge and build both?

File: rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec [hunk 1]
```diff
@@ -66,7 +66,7 @@ cp -pa .%{gem_instdir}/settings.d/pulpnode.yml.example %{buildroot}%{foreman_pro
 %{gem_instdir}/bundler.d
 %{gem_instdir}/settings.d
 %{foreman_proxy_bundlerd_dir}/pulp.rb
-%{_sysconfdir}/foreman-proxy/settings.d/pulp.yml
+%config(noreplace) %{_sysconfdir}/foreman-proxy/settings.d/pulp.yml
 %{_sysconfdir}/foreman-proxy/settings.d/pulpnode.yml
```
Comment (domcleal) [inline rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec]: This too.

**Comment by domcleal** [comment]
@ares that'd be wonderful, once that's fixed, thanks!

**Pull request merged**